jerry-core/config.h

```diff
@@ -180,4 +180,12 @@
  */
 #define CONFIG_PARSER_ENABLE_PARSE_TIME_BYTE_CODE_OPTIMIZER
 
+/**
+  * Enable timezone and daylightsaving system calls for date object
+  *
+  * Note:
+  *      Experimental. Works on Ubuntu 14.04.
+  */
+// #define ECMA_ENABLE_DATE_SYS_CALLS
```

Comment:
It looked like to me that is the common way. (see: CONFIG_VM_RUN_GC_AFTER_EACH_OPCODE) But I agree with you. A build option would be good.